contracts/axelar-gateway/src/contract.rs

```diff
@@ -8,6 +8,8 @@ use crate::storage_types::{DataKey, MessageApprovalKey, MessageApprovalValue};
 use crate::{auth, event};
 
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+pub const INSTANCE_TTL_THRESHOLD: u32 = 120_960; // approx 7 days
```

Comment:
add comment on why these params were chosen (how it relates to cross-chain message frequency etc)

Reply:
Added. Please review comment for clarity and readability.